react/components/CreditCard.css

```
@@ -0,0 +1,7 @@
+/* Using min-width to set the width of the iFrame, works around an issue in iOS that can 
+prevent the iFrame from sizing correctly. style={{ minWidth: '100%', minHeight: '200px' }} */
+
+.iframe {
+    min-width: 100%;
+    min-height: 200px;
```

nitpick the tab size
```suggestion
  min-width: 100%;
  min-height: 200px;
```